OPCB/functions/missions/spawn_city_rus.sqf

```diff
@@ -1,30 +1,23 @@
+params ["_centerobj"];
 
-	_centerobj = _this select 0;
+_houses = nearestObjects [_centerobj, ["house"], 500] select { count ( _x buildingPos -1 ) > 2 };
 
-	_houses = nearestObjects [_centerobj, ["house"], 500] select { count ( _x buildingPos -1 ) > 2 };
+_defendergroup = creategroup east;
 
-	_defendergroup = creategroup east;
+for "_i" from 0 to count _houses -1 do {
```

Comment:
If `count _houses == 0` this is going to error out but I don't know if there's any _real_ chance of that happening...

Reply:
I have also thought of that, but the chances are so slim I have decided to ignore it. The workaround would be far more work.

Reply:
Honestly, you could at least not go into that `for` at all ìf (count _houses == 0)` to at least avoid an unhandled error